docs/guides/vault_secrets_manager.md

```diff
@@ -0,0 +1,98 @@
+---
+page_title: "Using Vault Provider with STACKIT Secrets Manager"
+subcategory: ""
+description: |-
+   Guide on how to write secrets from Terraform deployed resources on STACKIT cloud to the STACKIT Secrets Manager 
+---
+# Using Vault Provider with STACKIT Secrets Manager
+
+### Overview
+
+This guide outlines the process of utilizing the HashiCorp Vault provider alongside the STACKIT provider to write secrets into the STACKIT Secrets Manager. Due to the current limitation in the STACKIT Terraform provider where direct writing of secrets isn't supported, this workaround is necessary.
```

Comment:
I wouldnt say this is a limitation of the provider. And this is not really a workaround. 
In general, the STACKIT services are providing control-plane APIs, that don't support performing operations in the instances themselves such as `write` operations. It's the same for writing stuff into an Object Storage bucket: you need to use the `s3` provider

So I would rephrase this

Reply:
This would be a workaround if the Secrets Manager would plan to add this functionality to their API but I think this will not happen